PCbuild/python.props

```diff
@@ -201,16 +201,16 @@
     ))</Field3Value>
     <Field3Value Condition="$(UseTestMarker) == 'true'">$([msbuild]::Add($(Field3Value), 9000))</Field3Value>
     
-    <!-- The name of the resulting pythonXY.dll (without the extension) -->
-    <PyDllName>python$(MajorVersionNumber)$(MinorVersionNumber)$(PyDebugExt)</PyDllName>
+    <!-- The name of the resulting pythonX_Y.dll (without the extension) -->
+    <PyDllName>python$(MajorVersionNumber)_$(MinorVersionNumber)$(PyDebugExt)</PyDllName>
```

Comment:
There are going to be a number of similar changes in Tools/msi

Reply:
Likely some under Tools/nuget as well, though they may all be under Tools/msi or PC/layout.

Reply:
grepped and changed where needed (I think)